apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2.java

```diff
@@ -273,6 +281,11 @@ private static ExplanationOfBenefit transformClaim(
                       TransformerUtilsV2.createExtensionCoding(
                           eob, CcwCodebookVariable.CARR_LINE_MTUS_CNT, code)));
 
+      String drugCode = null;
+
+      if (line.getNationalDrugCode().isPresent()) {
+        drugCode = drugCodeProvider.retrieveFDADrugCodeDisplay(line.getNationalDrugCode().get());
+      }
```

Comment:
I suggest encapsulating this in a method that can be reused across these transformers and invoking it inline below where drugCode is needed.

Reply:
changed it to take an optional...